process/process.go

```diff
@@ -248,9 +248,13 @@ func (p *Process) Run() error {
 		}
 	}
 
-	// Find the exit status of the script
-	p.logger.Info("Process with PID: %d finished with Exit Status: %d",
-		p.pid, p.status.ExitStatus())
+	// Find the exit status or terminating signal of the script
+	exitSignal := "nil"
```

Comment:
Perhaps the default could be "?" so folks know that we didn't know, as apposed to it just being blank?

Reply:
It isn’t that we don’t know, it’s that there isn’t one.